README.md

```diff
@@ -1,124 +1,25 @@
-`UnicodeDecodeError` from WebOb:
-
-```terminal
-curl 'http://localhost:6543/foo?q=%FC'
-```
-
-`pyramid-sanity` catches this and turns it into a 400 Bad Request response.
-
-Related issues:
-
-* https://github.com/Pylons/pyramid/issues/3399
-* https://github.com/Pylons/webob/issues/161
-
-</details>
-
-<details>
-<summary>A request with a badly encoded path can crash an app.</summary>
-
-With the example app above this will raise an uncaught
-`pyramid.exceptions.URLDecodeError`:
-
-```terminal
-curl 'http://localhost:6543/%FC'
-```
-
-`pyramid-sanity` catches this and turns it into a 400 Bad Request response.
-
-The 500 Server Error here is actually deliberate behavior from Pyramid:
-Pyramid raises [`URLDecodeError`](https://docs.pylonsproject.org/projects/pyramid/en/latest/api/exceptions.html#pyramid.exceptions.URLDecodeError)
-and does not have a built-in exception view for `URLDecodeError`. Apps can
-change this to a 400 Bad Request by registering a custom exception view for
-`URLDecodeError`. See <https://github.com/Pylons/pyramid/issues/312#issuecomment-2322368>.
-
-Related issues:
-
-* https://github.com/Pylons/pyramid/issues/434
-* https://github.com/Pylons/pyramid/issues/1374
-* https://github.com/Pylons/pyramid/issues/2047
-* https://github.com/Pylons/webob/issues/114
-* https://github.com/hypothesis/h/issues/4915
-* https://github.com/hypothesis/h/pull/4916
-
-</details>
-
-<details>
-<summary>A bad form submission can crash an app when the app tries to access <code>request.POST</code>.</summary>
-
-With the example app above this will raise an uncaught `ValueError` from WebOb:
-
-```terminal
-curl --request POST --url http://localhost:6543/foo --header 'content-type: multipart/form-data'
-```
-
-`pyramid-sanity` catches this and turns it into a 400 Bad Request response.
-
-Related issues:
+`pyramid-sanity` is a Pyramid extension that catches certain crashes caused by
+badly formed requests, turning them into `400: Bad Request` responses instead.
 
-* https://github.com/Pylons/pyramid/issues/1258
-
-</details>
-
-<details>
-<summary>If an app returns a redirect with a badly encoded location it can crash the WSGI server.</summary>
-
-With the example app above this will raise an uncaught `AttributeError` from `wsgiref.simple_server`:
-
-```terminal
-curl http://localhost:6543/redirect
-```
+It also prevents apps from returning HTTP redirects with badly encoded locations
+that can crash WSGI servers.
 
-`pyramid-sanity` intercepts this and safely encodes the redirect location.
+The aim is to have sensible defaults to make it easier to write a reliable Pyramid app.
 
-</details>
+For details of all the errors and fixes, and how to reproduce them see: [Error details](#error-details).
 
 Usage
 -----
 
 ```python
 with Configurator() as config:
+    config.add_settings({
+        # See the section below for all settings...        
+        "pyramid_sanity.check_form": False,
+    })
+    
```

Comment:
Added the config in situ so we don't have to repeat this in all the examples below